src/Compilers/VisualBasic/Portable/Scanner/Scanner.vb

```diff
@@ -2000,7 +1953,7 @@ FullWidthRepeat2:
             Dim YearIsTwoDigits As Boolean = False
             Dim DaysToMonth As Integer() = Nothing
             Dim yearIsFirst As Boolean = False
-
+            Dim ch, cx As Char, sep1 As Char
```

Comment:
Having ch, cx, and sep1 used heavily hurts readability.  Can these have better names?